numpy/polynomial/polynomial.py

```diff
@@ -137,9 +137,9 @@ def polyline(off, scl):
     Examples
     --------
     >>> from numpy.polynomial import polynomial as P
-    >>> P.polyline(1,-1)
+    >>> P.polyline(1, -1)
     array([ 1, -1])
-    >>> P.polyval(1, P.polyline(1,-1)) # should be 0
+    >>> P.polyval(1, P.polyline(1, -1)) # should be 0
```

Comment:
While we're fixing PEP8 whitespace,
```suggestion
    >>> P.polyval(1, P.polyline(1, -1))  # should be 0
```
[Inline comments should be separated by at least two spaces from the statement.](https://peps.python.org/pep-0008/#inline-comments)

Reply:
I changed this at a buch of other locations as well.